docs/docs/Develop/Clients/typescript-client.md

```diff
@@ -0,0 +1,268 @@
+```tsx
+const response = await client.flow(flowId).stream(input);
+
+for await (const event of response) {
+  console.log(event);
+}
+```
+4. Run the completed TypeScript application to call your server with `tweaks` and `session_id`, and stream the response back.
+Replace `baseUrl` and `flowId` with values from your deployment.
+
+<TabItem value="TypeScript" label="TypeScript" default>
+
+```tsx
+import { LangflowClient } from "@datastax/langflow-client";
+
+const baseUrl = "http://127.0.0.1:7860";
+const client = new LangflowClient({ baseUrl });
+
+async function runFlow() {
+    const flowId = "aa5a238b-02c0-4f03-bc5c-cc3a83335cdf";
+    const input = "Is anyone there?";
+    const tweaks = { model_name: "gpt-4o-mini" };
+    const session_id = "test-session";
+
+    const response = await client.flow(flowId).stream(input, {
+        session_id,
+        tweaks,
+      });
+
+    for await (const event of response) {
+        console.log(event);
+    }
+
+}
+runFlow().catch(console.error);
+```
+</TabItem>
+
+<details>
+<summary>Response</summary>
+
+```text
+{
+  event: 'add_message',
+  data: {
+    timestamp: '2025-05-23 15:52:48 UTC',
+    sender: 'User',
+    sender_name: 'User',
+    session_id: 'test-session',
+    text: 'Is anyone there?',
+    files: [],
+    error: false,
+    edit: false,
+    properties: {
+      text_color: '',
+      background_color: '',
+      edited: false,
+      source: [Object],
+      icon: '',
+      allow_markdown: false,
+      positive_feedback: null,
+      state: 'complete',
+      targets: []
+    },
+    category: 'message',
+    content_blocks: [],
+    id: '7f096715-3f2d-4d84-88d6-5e2f76bf3fbe',
+    flow_id: 'aa5a238b-02c0-4f03-bc5c-cc3a83335cdf',
+    duration: null
+  }
+}
+{
+  event: 'token',
+  data: {
+    chunk: 'Absolutely',
+    id: 'c5a99314-6b23-488b-84e2-038aa3e87fb5',
+    timestamp: '2025-05-23 15:52:48 UTC'
+  }
+}
+{
+  event: 'token',
+  data: {
+    chunk: ',',
+    id: 'c5a99314-6b23-488b-84e2-038aa3e87fb5',
+    timestamp: '2025-05-23 15:52:48 UTC'
+  }
+}
+{
+  event: 'token',
+  data: {
+    chunk: " I'm",
+    id: 'c5a99314-6b23-488b-84e2-038aa3e87fb5',
+    timestamp: '2025-05-23 15:52:48 UTC'
+  }
+}
+{
+  event: 'token',
+  data: {
+    chunk: ' here',
+    id: 'c5a99314-6b23-488b-84e2-038aa3e87fb5',
+    timestamp: '2025-05-23 15:52:48 UTC'
+  }
+}
+
+// this response is abbreviated
+
+{
+  event: 'end',
+  data: { result: { session_id: 'test-session', outputs: [Array] } }
+}
+```
+
+</details>
+
+## Retrieve logs from Langflow
+
+To retrieve Langflow logs, you must enable log retrieval on your Langflow server by including these values in your server's `.env` file:
+
+```text
+LANGFLOW_ENABLE_LOG_RETRIEVAL=true
+LANGFLOW_LOG_RETRIEVER_BUFFER_SIZE=10000
+LANGFLOW_LOG_LEVEL=DEBUG
+```
+
+For more information, see [API examples](/api-reference-api-examples#logs).
```

Comment:
This section feels incomplete at the moment.

Reply:
Example is updated with the client for 0.3.0